ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/impl/inmemory/ResourceAnnotationValidation.scala

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.index.impl.inmemory
+
+import java.nio.charset.StandardCharsets
+
+import scala.util.matching.Regex
+
+object ResourceAnnotationValidation {
```

Comment:
Can we merge the 2 `ResourceAnnotationValidation` into one?

Reply:
Yes. The other was the one a missed to delete.

FYI We have to have `ResourceAnnotationValidation` living in participant-state-index because, because it's used by in-memory stores that live in participant-state-index.